LLMonFHIR/FHIR Display/OpenAIChatView.swift

```diff
@@ -87,4 +77,109 @@ struct OpenAIChatView: View {
             }
         }
     }
+    
+    private func processMultipleResourceChat() async throws {
+        let resourcesArray = await fhirStandard.resources
+        var stringResourcesArray = resourcesArray.map { "\($0.displayName) in \($0.resourceType)" }
+        stringResourcesArray.append("N/A")
+        let functionCallOutputArray = try await getFunctionCallOutputArray(stringResourcesArray)
+        
+        if !functionCallOutputArray.contains(where: { $0 == "N/A" }) && !functionCallOutputArray.contains(where: { $0 == " N/A" }) {
```

Comment:
You can reduce this down to a statement like this, which just checks if "N/A" is somewhere in the string that you look for:
```suggestion
        if !functionCallOutputArray.contains(where: { $0.contains("N/A") }) {
```